util/util.js

```diff
@@ -0,0 +1,9 @@
+class Util {
+  constructor() {
+    throw new Error("This class may not be initiated with new");
+  }
+}
+
+Util.wait = require("util").promisify(setTimeout);
```

Comment:
if you make a class then why you add a method that way, instead actuall add it inside the class

Reply:
Because its a static method.